content/tutorials/aws-persistence.textile

```diff
@@ -0,0 +1,305 @@
+  
+5. Select the region your AWS account is located in.
+
+6. Enter @ably-persistence@ as the Function Name.
+
+7. Enter your AWS Credentials. This must be in the format @AccessKeyID:SecretKeyID@. For example, @AKIAUUFPRLMPNUEMFQPK:hGdpSuMaEthcNtxH/KMdyDsrjxYj53sDwWuvjm3p@.
+
+8. Select *JSON* as the Encoding option.
+ 
+9. Uncheck *Enveloped* (to keep the tutorial as simple as possible).
+
+10. Click the *Create* button.
+
+h2(#create-ably-page). Create a user interface
+
+To start generating messages to supply to your webhook you will set up a basic HTML page. This page will subscribe to a channel and publish messages to that channel when the user clicks a button.
+
+Create a new @index.html@ file on your local machine with the following contents:
+
+```[html]
+<!DOCTYPE html>
+<html lang="en">
+  <body>
+    <h1>Ably Webhook Lambda Test</h1>
+    <button id="publish" type="button" value="Connecting...">Click to Publish</button>
+    <div id="result" style="padding: 10px"></div>
+  </body>
+
+  <script
+    type="text/javascript"
+    src="https://cdn.ably.io/lib/ably.min-1.js"
+  ></script>
+  <!-- Include the latest Ably Library  -->
+
+  <!-- Instance the Ably library  -->
+  <script>
+    const ably = new Ably.Realtime("ABLY_API_KEY"); /* ADD YOUR API KEY HERE */
+    channel =
+      ably.channels.get(
+        "persistencetest"
+      ); /* Channel name being published and subscribed to */
+
+    const generateRandomString = (length = 8) =>
+      Math.random().toString(20).substr(2, length);
+
+    const btn = document.getElementById("publish");
+    const results = document.getElementById("result");
+
+    btn.addEventListener("click", () => {
+      channel.publish("message", `${generateRandomString()}`);
+    });
+
+    ably.connection.on("connected", () => {
+      btn.innerText = "Connected to Ably, click to publish a message";
+    });
+
+    channel.subscribe((message) => {
+      results.innerHTML += `Message from Ably: ${message.data}<br/>`;
+    });
+  </script>
+</html>
+```
+
+This file displays a simple interface for testing your webhook. It uses the Ably Realtime SDL to publish messages to and subscribe to messages from the @persistencetest@ channel.
+
+Ensure that you replace @ABLY_API_KEY@ with your own app's API key from the "Ably dashboard":https://ably.com/accounts.
+
+Open @index.html@, and when the button caption states that you are connected to Ably, click it. A new, random message is sent to the channel. You can further inspect the connections made to and messages sent from your app by viewing the "Stats" tab for your app in the "Ably dashboard":https://ably.com/accounts.
+
+If you now navigate to your "S3 Buckets":https://s3.console.aws.amazon.com/s3/buckets and select the @ably-message-data@ bucket. Click the @test.txt@ link and then click the *Open* button. Your browser will download the @test.txt@ file. Open the file in a text editor and verify that it contains the list of messages sent.
+
```

Comment:
It would have been nice to have some concluding text/summary here to confirm that everything has successfully been carried out.